kotlin/internal/opts.bzl

```diff
@@ -49,6 +49,27 @@ _KOPTS = {
         ),
         type = attr.bool,
     ),
+     "x_inline_classes": struct(
+        args = dict(
+            default = False,
+            doc = "Enable experimental inline classes",
+        ),
+        type = attr.bool,
+    ),
+    "x_allow_result_return_type": struct(
+        args = dict(
+            default = False,
+            doc = "Enable kotlin.Result as a return type",
+        ),
+        type = attr.bool,
+    ),
+    "x_opt_in": struct(
```

Comment:
Drop x_opt_in for this pr? It's been removed form compile.

Reply:
Sorry - missed that one

Reply:
Apologies for the delay.